From 93ff8640baaeca61161d2a86ef1db54cb965ba72 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Thu, 2 Sep 2010 18:05:49 +0000
Subject: [PATCH 11/25] gettextize: git-merge "In-index merge" message
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Make the "In-index merge" message translatable. This just emits to
stdout via finish(), so translating it should be fine.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 builtin/merge.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/builtin/merge.c b/builtin/merge.c
index bf3b313..956ce14 100644
--- a/builtin/merge.c
+++ b/builtin/merge.c
@@ -791,7 +791,7 @@ static int merge_trivial(void)
 	parent->next->item = remoteheads->item;
 	parent->next->next = NULL;
 	commit_tree(merge_msg.buf, result_tree, parent, result_commit, NULL);
-	finish(result_commit, "In-index merge");
+	finish(result_commit, _("In-index merge"));
 	drop_save();
 	return 0;
 }
-- 
1.7.2.2.614.g1dc9

